lib/matplotlib/backends/backend_wx.py

```diff
@@ -318,7 +318,7 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
         if angle == 0.0:
             gfx_ctx.DrawText(s, x, y)
         else:
-            rads = angle / 180.0 * math.pi
+            rads = math.radians(angle)
```

Comment:
Oops, I meant to go back and check whether that was `math` or `np` and forgot about it.